### PR TITLE
[1LP][RFR] Add test_fixauth_dryrun_has_feedback

### PR DESCRIPTION
**Pull request by jaryn**
Add test for BZ 1577303

### CODE REVIEW

File: cfme/tests/test_appliance.py [hunk 1]
```diff
@@ -435,3 +435,24 @@ def codename_in_stdout():
...
+        casecomponent: Appliance
+        initialEstimate: 1/60h
+
+    Testing BZ 1577303
+    """
+    appliance = temp_appliance_preconfig
+    run_command = appliance.ssh_client.run_command
+    dry_run_message = (
+        'is executing in dry-run mode, and no actual changes will be made **')
+    assert dry_run_message in run_command("fix_auth -d").output
```
Comment (digitronik) [inline cfme/tests/test_appliance.py]: optional:
we can combine this 4 assertions but totally optional

Comment (jaryn) [reply cfme/tests/test_appliance.py]: I don't like to combine them. I don't see a reason for doing so. I see just a reason for _not_ doing so. When they are separate, one can more easily see which one failed.

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

File: cfme/tests/test_appliance.py [hunk 2]
```diff
@@ -435,3 +435,24 @@ def codename_in_stdout():
...
+def test_fixauth_dryrun_has_feedback(temp_appliance_preconfig):
+    """
+    Check whether the fixauth says it is running in dry mode
+
+    Polarion
+        assignee: jhenner
+        casecomponent: Appliance
+        initialEstimate: 1/60h
+
+    Testing BZ 1577303
```
Comment (digitronik) [inline cfme/tests/test_appliance.py]: I missed this... @jaryn we started Bugzilla collection consistency under a common tag like 

https://github.com/ManageIQ/integration_tests/blob/c5cba3736dea0110a5d01a6d345d61214f4f6102/cfme/tests/automate/test_domain.py#L273

Also if you are automating you and add `automate` marker.

**Pull request merged**